src/components/BreadCrumb/BreadCrumb.test.tsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import '@testing-library/jest-dom';
+import BreadCrumb from './BreadCrumb';
+import { BreadCrumbProps } from './BreadCrumb';
+import { render, screen, fireEvent } from '@testing-library/react';
+
+describe('BreadCrumb Component', () => {
+  const renderComponent = (props: BreadCrumbProps) =>
+    render(<BreadCrumb {...props} />);
+
+  test('renders all breadcrumb items', () => {
+    const items = [
+      { label: 'Home', href: '/' },
+      { label: 'Services', href: '/services' },
+      { label: 'Plumbing', href: '/services/plumbing' },
+    ];
+
+    renderComponent({ items });
+
+    items.forEach((item) => {
+      expect(screen.getByText(item.label)).toBeInTheDocument();
+    });
+  });
+
+  test('highlights the last breadcrumb item as bold', () => {
```

Comment:
can a test case also be included where the last breadcrumb item should not be clickable ?